Library/Homebrew/cmd/test-bot.rb

```diff
@@ -532,7 +531,6 @@ def formula(formula_name)
       test "brew", "fetch", "--retry", *formula_fetch_options
       test "brew", "uninstall", "--force", canonical_formula_name if formula.installed?
       install_args = []
-      install_args << "--verbose" unless ARGV.include? "--no-verbose-install"
```

Comment:
I almost forget that we still need to print something for travis. Otherwise, it will be killed if no output in long time.